exporters/fluentd/include/opentelemetry/exporters/fluentd/log/recordable.h

```diff
@@ -0,0 +1,123 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#define ENABLE_LOGS_PREVIEW 1
+
+#include "nlohmann/json.hpp"
+#include "opentelemetry/sdk/common/attribute_utils.h"
+#include "opentelemetry/sdk/logs/recordable.h"
+#include "opentelemetry/version.h"
+
+#include "opentelemetry/exporters/fluentd/common/fluentd_common.h"
+#include "opentelemetry/exporters/fluentd/common/fluentd_fields.h"
+
+#include <chrono>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter {
+namespace fluentd {
+namespace logs {
+using FluentdLog = nlohmann::json;
+
+static inline nlohmann::byte_container_with_subtype<std::vector<std::uint8_t>>
+get_msgpack_eventtimeext(int32_t seconds = 0, int32_t nanoseconds = 0) {
```

Comment:
Does this function also exist in fluentd::common namespace?

Reply:
Nice catch. Forgot to remove it from here after moving to fluentd::common namespace. Fixed now.